readme.md

```diff
@@ -1 +1,59 @@
-# gravitational
+# Scheduler-Worker Cluster Architecture
+
+
+# Goal
+
+*   Build a scheduler and worker architecture that supports GRPC communication.
+
+# General Overview
+
+*   Support single scheduler and multiple workers.
+*   The scheduler must be aware of all workers' states. (requires the workers to register and deregister when going offline)
```

Comment:
Do you think it can be guaranteed, that the workers will properly deregister when going offline?

Reply:
Yes, as long as there wasnt an unexpected error. I will implement a fatal function that calls the deregister function before killing the app